gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/publisher/HiveConvertPublisher.java

```diff
@@ -79,6 +88,15 @@
   private EventSubmitter eventSubmitter;
   private final FileSystem fs;
   private final HiveSourceWatermarker watermarker;
+  private final HiveMetastoreClientPool pool;
+
+  public static final String PARTITION_PARAMETERS_WHITELIST = "hive.conversion.partitionParameters.whitelist";
```

Comment:
How do you control dataset level whitelist / blacklist for this? This seems to read from WU - which seems to be coming from pull file properties, how about enabling config-store for this? 

nit: keys can go to AvroToORC config key class